utils/docker/Dockerfile.jammy

```diff
@@ -4,6 +4,8 @@ ARG DEBIAN_FRONTEND=noninteractive
 ARG TZ=America/Los_Angeles
 ARG DOCKER_IMAGE_NAME_TEMPLATE="mcr.microsoft.com/playwright:v%version%-jammy"
 
+ENV LANG=C.UTF-8
```

Comment:
Could this breaking in some situations? Let's make sure we have this in the miscellaneous list in the release notes.